config/sheets.json

```diff
@@ -230,11 +230,11 @@
         },
         "note": {
           "en": "(Optional) The access methods and publication formats via which the data in the data source is accessed, e.g. public JSON API.",
-          "es": "(Optional) The access methods and publication formats via which the data in the data source is accessed, e.g. public JSON API."
+          "es": "(Opcional) Los métodos de acceso y formatos de publicación a través de los cuales se accede a los datos de la fuente de datos, por ejemplo, la API JSON pública."
```

Comment:
In here the translation of public JSON API is
La API JSON pública
but just below it is translated as
API publica con JSON

There should be consistency.